modules/getent/tfgetent.c

```diff
@@ -77,6 +77,13 @@ _getent_format_array(gchar *name_or_gid, gpointer members, GString *result)
   return TRUE;
 }
 
+static gboolean
+_getent_format_array_at_location(gchar *name_or_gid, gpointer location, GString *result)
```

Comment:
Why the new function is needed ?
The original `_getent_format_array` is not used anywhere else.